ecosystem/sep-0012.md

```diff
@@ -315,3 +316,38 @@ Situation | Response
 Success | `200 OK`
 Client not authenticated properly | `401 Unauthorized`
 Anchor has no information on the customer | `404 Not Found`
+
+## Customer callback PUT
+
+Allow the wallet to provide a callback URL to the anchor. The provided callback 
+URL will replace (and supercede) any previously-set callback URL for this account.
+
+Whenever the user's
+`status` field changes, the anchor will issue a POST request to the callback
+URL. The payload of the POST request will be the same as [`GET /customer`](#customer-get).
+
+Anchors will submit POST requests until the user's status changes to `ACCEPTED`
+or `REJECTED`. If a wallet needs to watch a user's KYC status after that,
+it will need to set a callback again.
+
+### Request
```

Comment:
If a callback request to the provided URL fails, is the server obligated to retry? I don't think it is, but we should make that clear in this section.